responsibleai/environments/responsibleai-text-ubuntu20.04-py38-cpu/context/conda_dependencies.yaml

```diff
@@ -32,4 +32,4 @@ dependencies:
     - click<8.0.0
     - mltable==0.1.0b4
     - transformers>=4.17.0
-    - ml-wrappers==0.2.2
\ No newline at end of file
+    - ml-wrappers==0.2.2
```

Comment:
can you update ml-wrappers to latest pypi 0.4.2?

Reply:
Updated!